src/engine/Server.cpp

```diff
@@ -424,7 +424,7 @@ string Server::composeResponseJson(const string& query,
 
   string msg = ad_utility::escapeForJson(exception.what());
 
-  os << "\"Exception-Error-Message\": \"" << msg << "\"\n"
+  os << "\"exception\": \"" << msg << "\"\n"
```

Comment:
Do the QLever UI folks know about this change already?

Reply:
I hadn't realized this could cause problems for them. I changed the name to be consistent with the `ad_semsearch::Exception` naming, as the old ui does not print the error message otherwise.